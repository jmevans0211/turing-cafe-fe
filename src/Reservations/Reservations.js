import React from 'react';
// import PropTypes from 'prop-types';
import './Reservations.css';
import Card from '../Card/Card';

const Reservations = props => {
  const reservedTables = props.reservations.map(reserved => {
    return (<Card 
      id = {reserved.id}
      name = {reserved.name}
      date = {reserved.date}
      time = {reserved.time}
      number = {reserved.number}
    />)
  })
  return (
    <main className="reservations">
      {reservedTables}
    </main>
  )
}

export default Reservations;