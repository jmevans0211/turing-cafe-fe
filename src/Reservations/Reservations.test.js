import React from 'react';
import { shallow } from 'enzyme'
import Reservations from './Reservations';

describe('Reservations', () => {
  let wrapper
  // const mockFn = jest.fn();

  beforeEach(() => {
    wrapper = shallow(<Reservations reservations= {
      [
        {
          id: 18907224, 
          name: 'Christie', 
          date: '8/8', 
          time: '7:00', 
          number: 3 
        }
      ]
    }
    />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

 })