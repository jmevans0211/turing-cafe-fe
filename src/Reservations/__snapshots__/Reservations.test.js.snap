// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reservations should match the snapshot 1`] = `
<main
  className="reservations"
>
  <Card
    date="8/8"
    id={18907224}
    name="Christie"
    number={3}
    time="7:00"
  />
</main>
`;
